src/libs/ajax.js

```diff
@@ -51,11 +52,20 @@ const withCancellation = wrappedFetch => async (...args) => {
   }
 }
 
+const captureRetryFailure = (...args) => {
```

Comment:
The name of this function suggests that it is used to capture failures of retries. In reality, it's used to capture the first failure of a request. Failures in retries are not captured (since this function is wrapped in `_.once`).

```suggestion
const captureRequestFailure = (...args) => {
```